packages/components/psammead-caption/package.json

```diff
@@ -17,8 +17,9 @@
   },
   "homepage": "https://github.com/bbc/psammead/blob/latest/packages/components/psammead-caption/README.md",
   "dependencies": {
-    "@bbc/gel-foundations": "^0.2.5",
-    "@bbc/psammead-styles": "^0.3.2"
+    "@bbc/gel-foundations": "^1.0.0",
+    "@bbc/psammead-styles": "^0.3.2",
+    "styled-components": "^4.1.2"
```

Comment:
Looks like a merge issue, I think styled components are in devDependencies only now